src/components/Header/Header/index.tsx

```diff
@@ -1,13 +1,14 @@
-import './Header.css'
 import { HeaderLeft } from '../HeaderLeft/'
 import { HeaderRight } from '../HeaderRight'
 import { type FC } from 'react'
+import { HeaderWrapper } from './styles'
+import headerBackImg from '../../../thumbnails/patterns/darkBluePattern.jpg'
 
 export const Header: FC = () => {
   return (
-    <div className="header">
+    <HeaderWrapper HeaderBackground={headerBackImg}>
```

Comment:
NIT (this mean nitpick which is like not a horrible comment, but a small nitpicky comment).
anyways NIT: variable names should be camelcase